src/font.css

```
@@ -0,0 +1,9 @@
+@font-face {
+    font-family: 'RussoOne';
+    src: url('./fonts/Russo_One/RussoOne-Regular.ttf');
```

这里也可以在末尾加上`format('truetype')`
```suggestion
    src: url('./fonts/Russo_One/RussoOne-Regular.ttf') format('truetype');
```

这里写的`format('truetype')`意义在于，可以帮助浏览器甄选所需要的字体格式